extension/chrome/elements/passphrase.ts

```diff
@@ -42,6 +42,12 @@ View.run(class PassphraseView extends View {
     if (!this.orgRules.forbidStoringPassPhrase()) {
       $('.forget').prop('disabled', false);
     }
+    if (this.orgRules.usesKeyManager()) {
+      $('#lost-pass-phrase').removeAttr('id').attr('href','#');
```

Comment:
I think this way it will still appear clickable? You'd probably have to remove the `href` too. Or have two completely separate elements, one `a` (original) and one `div class=line` (updated) for example.

Finally, `pass-phrase-alt` is not so descriptive, better would be `lost-pass-phrase-with-ekm`